lib/Logger/Logger.h

```diff
@@ -7,22 +7,20 @@
 class Logger {
     public:
         enum class Level : int {
-            TRACE = 0,
-            DEBUG = 1,
-            INFO = 2,
+            TRACE   = 0,
+            DEBUG   = 1,
+            INFO    = 2,
             WARNING = 3,
-            ERROR = 4,
-            FATAL = 5,
-            SILENT = 6,
+            ERROR   = 4,
+            FATAL   = 5,
+            SILENT  = 6,
         };
 
         /**
          * @brief Return singleton instance of the logger
          * @return Logger instance
          */
-        static Logger& getInstance() {
-            return getInstanceImpl();
-        }
+        static Logger& getInstance() { return getInstanceImpl(); }
```

Comment:
as before, was fine before formatting

Reply:
Yeah, I'm not a fan. 
Return statements belong on a separate line.

Reply:
Aye, see above, switched off.